build.gradle

```diff
@@ -1,8 +1,15 @@
 plugins {
     id "java"
-    id "edu.wpi.first.GradleRIO" version "2024.3.2"
+    id "edu.wpi.first.GradleRIO" version "2025.1.1-beta-3"
     id("com.diffplug.spotless") version "7.0.0.BETA2"
-    id 'org.jetbrains.kotlin.jvm'
+    id 'org.jetbrains.kotlin.jvm' version "2.0.10"
+    /*
+    To setup epilogue,
+    uncomment the 'id "org.jetbrains.kotlin.kapt" version "2.0.21"' line below
+    and the 'kapt wpilib.java.deps.wpilibAnnotations()' line
+    in the 'dependencies' section
+     */
+    //id "org.jetbrains.kotlin.kapt" version "2.0.21"
```

Comment:
Why disable this by default?

Reply:
Kapt generates java "stubs" in kotlin, which means that initial compile times are slightly longer. However, subsequent compilations are about the same speed.
If teams don't use epilogue there's not much of a reason to use the generated stubs.

Reply:
Many teams avoid editing the build.gradle file like the devil, I'd rather its enabled by default if thats the only downside